spark/src/test/java/com/netflix/iceberg/spark/source/TestFilteredScan.java

```diff
@@ -22,12 +22,7 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Lists;
-import com.netflix.iceberg.DataFile;
-import com.netflix.iceberg.DataFiles;
-import com.netflix.iceberg.FileFormat;
-import com.netflix.iceberg.PartitionSpec;
-import com.netflix.iceberg.Schema;
-import com.netflix.iceberg.Table;
+import com.netflix.iceberg.*;
```

Comment:
Nit: Please don't use globbed imports. It makes it harder to follow imported symbols in github and simpler editors.

Reply:
Its my editor again. Will fix.